Flaky E2E test: test_search is flaky due to tag reported by karton.classifier

**Environment information**
* MWDB version (from `/about`):
* Installation method:
  - [ ] mwdb.cert.pl service
  - [ ] From PyPi (`pip install mwdb-core`)
  - [ ] From docker-compose
  - [x] Other (please explain)
* Plugins installed:

**Behaviour the bug (what happened?)**

```
mwdb-tests\_1 | 2023-01-18T10:53:59.663505243Z E AssertionError: assert \[{'tag': 'misc:ascii'}\] == \[\]

mwdb-tests\_1 | 2023-01-18T10:53:59.663508443Z E Left contains one more item: {'tag': 'misc:ascii'}

mwdb-tests\_1 | 2023-01-18T10:53:59.663511643Z E Full diff:

mwdb-tests\_1 | 2023-01-18T10:53:59.663514743Z E - \[\]

mwdb-tests\_1 | 2023-01-18T10:53:59.663517743Z E + \[{'tag': 'misc:ascii'}\]

mwdb-tests\_1 | 2023-01-18T10:53:59.663521043Z

mwdb-tests\_1 | 2023-01-18T10:53:59.663524143Z test\_api.py:87: AssertionError

mwdb-tests\_1 | 2023-01-18T10:53:59.663527243Z =========================== short test summary info ============================
```

https://github.com/CERT-Polska/mwdb-core/actions/runs/3948080257/jobs/6757702055

**Expected behaviour**

<!-- What did you expect to happen? -->

Test should pass.

--- a/tests/backend/test_api.py
+++ b/tests/backend/test_api.py
@@ -82,9 +82,6 @@ def test_search(admin_session):
         assert res['id'] == sample['id']
         assert res['file_name'] == filename
         assert res['file_size'] == len(file_content)
-        assert res['parents'] == []
-        assert res['children'] == []
-        assert res['tags'] == []
         parse(res['upload_time'])
 
 

